src/OVAL/probes/SEAP/_seap-scheme.h

```diff
@@ -26,7 +26,11 @@
 
 #include <stddef.h>
 #include <stdint.h>
+#ifdef _WIN32
```

Comment:
Just a remark - [stackoverflow](https://stackoverflow.com/questions/341817/is-there-a-replacement-for-unistd-h-for-windows-visual-c) also suggests that `process.h` also has some stuff as `unistd.h`.